### ISSUE TITLE
Docker Installed?

### ISSUE THREAD

**Issue by codepj** [GoogleCloudPlatform/jenkins-packer-agent#4]
I believe jessie no longer has docker included.  

Works by following these instrutions to install docker:
https://docs.docker.com/engine/installation/debian/

Updating the supervisor file to run docker.

Then running the entire thing with the --privileged=true flag

**Comment by evandbrown** [comment]
You're right, Docker isn't installed on Jessie. This image supports the [Jenkins on Kubernetes project](https://github.com/GoogleCloudPlatform/kube-jenkins-imager), and Docker-in-Docker was causing headaches for lots of folks, so we moved to using host Docker to do the builds: https://github.com/GoogleCloudPlatform/kube-jenkins-imager

I'll leave this issue open until I can update the instructions to point to that project and remove the dind mentions (probably tomorrow). Thanks for pointing this out!

**Comment by codepj** [comment]
NP, I found this repo via a link on the [Automated Image Builds with Jenkins, Packer, and Kubernetes solution paper](https://cloud.google.com/solutions/automated-build-images-with-jenkins-kubernetes).  I saw that the associated tutorial uses the "gcr.io/cloud-solutions-images/jenkins-packer-agent:master-e8758ac" image but wanted to try and build one myself.  Looking forward to the update!
